includes/class-wc-tax.php

```diff
@@ -36,6 +36,17 @@ public static function init() {
 		self::$round_at_subtotal = 'yes' === get_option( 'woocommerce_tax_round_at_subtotal' );
 	}
 
+	/**
+	 * When the woocommerce_tax_classes option is changed, remove any orphan rates.
+	 *
+	 * @deprecated 3.7.0
+	 * @param  string $old_value Old rates value.
+	 * @param  string $value New rates value.
+	 */
+	public static function maybe_remove_tax_class_rates( $old_value, $value ) {
+		wc_deprecated_function( 'WC_Tax::maybe_remove_tax_class_rates', '3.7' );
```

Comment:
It seems to me we should include the implementation here as well. Previously, this was a function that took 2 string parameters and based on their value updated `wp_woocommerce_tax_rates` and `wp_woocommerce_tax_rate_locations` db tables. It's still a breaking change if this function becomes a noop producing only a warning.